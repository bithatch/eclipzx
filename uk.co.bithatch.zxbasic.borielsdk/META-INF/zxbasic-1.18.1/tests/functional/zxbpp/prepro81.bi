#define SCREEN_W1 5

#if SCREEN_W == 5
   LET a$ = "test"
#endif
