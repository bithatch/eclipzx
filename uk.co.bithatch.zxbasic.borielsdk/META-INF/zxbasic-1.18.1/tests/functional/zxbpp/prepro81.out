#line 1 "prepro81.bi"


#line 6 "prepro81.bi"
